bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/ValueUtils.java

```diff
@@ -145,6 +145,14 @@ public static BObject createObjectValue(Module packageId, String objectTypeName,
         return objectValue;
     }
 
+    /**
+     * Create an empty array of Future values. (This helper method is currently used by the debugger expression
+     * evaluator).
+     */
+    public static FutureValue[] createFutureArray() {
```

Comment:
Both methods added to the runtime are used only by debugger runtime. Can we move them to debug-adapter:debug-adapter-runtime?

Reply:
@warunalakshitha I tested this locally and the evaluation returns an error wrapped inside `WaitResult` after the first evaluation. Since the expression evaluation is supposed to be dependent on the modified state of the JVM, I guess this behaviour should be expected from the user's POV. WDYT?

Reply:
@warunalakshitha `debug-adapter-runtime` module only has a ballerina package as of now. I'll try to move these java utils by creating a new module.